auth/src/main/res/layout/fui_phone_layout.xml

```diff
@@ -1,57 +1,53 @@
 <?xml version="1.0" encoding="utf-8"?>
 <ScrollView
     xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
     android:layout_width="match_parent"
     android:layout_height="match_parent">
 
-    <RelativeLayout
+    <android.support.constraint.ConstraintLayout
         style="@style/FirebaseUI.WrapperStyle"
-        android:layout_width="match_parent"
         android:layout_height="wrap_content">
 
         <com.firebase.ui.auth.ui.phone.CountryListSpinner
             android:id="@+id/country_list"
-            style="@style/FirebaseUI.CountrySpinner" />
+            style="@style/FirebaseUI.CountrySpinner"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintBottom_toBottomOf="@id/phone_layout" />
```

Comment:
Before this was baseline aligned, I don't think bottom and baseline are the same.

Reply:
Good catch! 😄 When an error appears, the spinner follows the error which is wrong. I have to play around to see how to fix that.